intellst-front/src/store/constants.js

```diff
@@ -0,0 +1,3 @@
+const BaseUrl = "http://localhost/";
```

Comment:
This shouldn't be hardcoded. It is better to place the configuration variable in environment or ignored file.

Reply:
It's moved to the config.js file which is ignored